convert2rhel/toolopts.py

```diff
@@ -300,6 +307,12 @@ def _process_cli_options(self):
             else:
                 tool_opts.serverurl = parsed_opts.serverurl
 
+        if parsed_opts.keep_rhsm:
+            if parsed_opts.disable_submgr:
```

Comment:
```suggestion
            if parsed_opts.no_rhsm:
```

Reply:
It should have been one of these two options:
- `if parsed_opts.no_rhsm or parsed_opts.disable_submgr:`
- `if tool_opts.no_rhsm:`

I've chosen the latter.